<testsuites id="" name="" tests="6" failures="2" skipped="2" errors="0" time="1.8361030000001193">
<testsuite name="tests/api/b.spec.ts" timestamp="2023-06-09T00:45:17.404Z" hostname="api" tests="6" failures="2" skipped="2" time="0.369" errors="0">
<testcase name="test_b_1 " classname="tests/api/b.spec.ts" time="0.067">
</testcase>
<testcase name="test_b_2" classname="tests/api/b.spec.ts" time="0.063">
</testcase>
<testcase name="test3" classname="tests/api/b.spec.ts" time="0.074">
<failure message="b.spec.ts:11:5 test3" type="FAILURE">
<![CDATA[  [api] › tests/api/b.spec.ts:11:5 › test3 ─────────────────────────────────────────────────────────

    Error: expect(received).toBeTruthy()

    Received: false

      10 |
      11 | test('test3', async ( ) => {
    > 12 |   expect(false).toBeTruthy();
         |                 ^
      13 | });
      14 |
      15 | test('test4', async ( ) => {

        at /Users/rk/Automation/test_env/tests/api/b.spec.ts:12:17
]]>
</failure>
</testcase>
<testcase name="test4" classname="tests/api/b.spec.ts" time="0.068">
<failure message="b.spec.ts:15:5 test4" type="FAILURE">
<![CDATA[  [api] › tests/api/b.spec.ts:15:5 › test4 ─────────────────────────────────────────────────────────

    Error: expect(received).toBeTruthy()

    Received: false

      14 |
      15 | test('test4', async ( ) => {
    > 16 |   expect(false).toBeTruthy();
         |                 ^
      17 | });
      18 |
      19 | test('test5', async ( ) => {

        at /Users/rk/Automation/test_env/tests/api/b.spec.ts:16:17
]]>
</failure>
</testcase>
<testcase name="test5" classname="tests/api/b.spec.ts" time="0.062">
<properties>
<property name="skip" value="">
</property>
</properties>
<skipped>
</skipped>
</testcase>
<testcase name="test6" classname="tests/api/b.spec.ts" time="0.035">
<properties>
<property name="skip" value="">
</property>
</properties>
<skipped>
</skipped>
</testcase>
</testsuite>
</testsuites>